libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1146,6 +1146,11 @@ bool TypeChecker::visit(FunctionCall const& _functionCall)
 	else
 		_functionCall.annotation().type = make_shared<TupleType>(functionType->returnParameterTypes());
 
+	if (functionType->bound() && !functionType->hasSelfType()) {
```

Comment:
`b.b()` will first cause `MemberAccess` and then `FunctionCall` visits, if the latter is an ICE then the whole thing is an ICE.